README.adoc

```diff
@@ -5,15 +5,16 @@ It is very easy to manually analyze a C, C++ and Objective-C project with SonarQ
 
 . Create a `sonar-project.properties` file to store your configuration
 .. Add a projectKey of your project
-.. Add a Build Wrapper output directory
 .. https://docs.sonarqube.org/latest/user-guide/user-token/[Generate a unique token] and set it as "sonar.login" (`SONARQUBE_TOKEN` in this example project)
 .. Set `sonar.host.url` to your SonarQube server URL
 . Download and unzip the Build Wrapper for macOS from {SonarQube URL}/static/cpp/build-wrapper-macosx-x86.zip
 . Add the Build Wrapper to your `$PATH`
 . Download and unzip the https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[SonarScanner] for macOS
 . Add the bin directory of the SonarScanner to your `$PATH`
-. Wrap your clean build command with the Build Wrapper. E.g.: `build-wrapper-macosx-x86 --out-dir bw-output xcodebuild clean build` (with `bw-output` being the same directory as the one set in `sonar-project.properties`)
-. Run `sonar-scanner`
+. Wrap your clean build command with the Build Wrapper. E.g.: `build-wrapper-macosx-x86 --out-dir bw-output xcodebuild clean build`
+. Run `sonar-scanner` specifying 
+.. `-Dsonar.cfamily.compile-commands=bw-output/compile_commands.json` (with `bw-output` being the same directory as the one passed to Build Wrapper as `--out-dir`) if you are using SonarQube version 10.6 or later
```

Comment:
Move `(with bw-output being the same directory as the one passed to Build Wrapper as --out-dir)` to the Run `sonar-scanner` line. Something like:

> Run `sonar-scanner` and specify one of the following properties (assuming `bw-output` is the same directory as the one passed to Build Wrapper as `--out-dir`):
>  - Either, ...
>  - Or, ...

WDYT?

Reply:
Can we put the assuming that `bw-output` is the output folder of build wrapper after both? to avoid having too much stuff before the continuation of the command?

I say this because we are already passing `bw-output` to build-wrapper in the previous step, it's also kind of intuitive that they should be the same, I'd rather put the clarification after than before

Reply:
Makes sense!

Reply:
Actually we cannot, we are in a numbered bulletpoint list, it would look weird even if we manage to do it. I will put it as you suggested